lib/handler/status/durations.c

```diff
@@ -125,11 +138,21 @@ static h2o_iovec_t durations_status_final(void *priv, h2o_globalconf_t *gconf, h
             "request-total-time") "," DURATION_FMT("process-time") "," DURATION_FMT("response-time") "," DURATION_FMT("duration"),
         DURATION_VALS(connect_time), DURATION_VALS(header_time), DURATION_VALS(body_time), DURATION_VALS(request_total_time),
         DURATION_VALS(process_time), DURATION_VALS(response_time), DURATION_VALS(total_time));
-
-#undef BUFSIZE
 #undef DURATION_FMT
 #undef DURATION_VALS
-
+    char *delim = "";
+    ret.len += sprintf(ret.base + ret.len, ",\n\"evloop-latency-nanosec\": [");
+    for(int i = 0; i < agg_stats->stats.evloop_latency_nanosec.size; i++) {
+        size_t len = snprintf(NULL, 0, "%s%llu", delim, agg_stats->stats.evloop_latency_nanosec.entries[i]);
```

Comment:
s/llu/PRIu64/? I believe that the argument types are uint64_t. Applies to the sprintf below too.